pages/common/ibmcloud.md

```diff
@@ -0,0 +1,24 @@
+# ibmcloud
+
+> A command line tool for managing IBM Cloud apps and services.
+> More information: <https://cloud.ibm.com/docs/cli?topic=cli-ibmcloud_cli>.
+
+- Update `ibmcloud` to the latest version:
+
+`ibmcloud update`
+
+- Install the Cloud Foundry module for accessing Cloud Foundry services:
+
+`ibmcloud cf install`
+
+- List all available IBM Cloud regions:
+
+`ibmcloud regions`
+
+- Show `ibmcloud` version:
+
+`ibmcloud version`
+
+- Show help for a specific subcommand (use without subcommand for general help):
```

Comment:
I like the idea to split this into `Display help` and `Display help for a subcommand`

Reply:
Yeah, I was thinking about this as well, I just thought it might be too verbose.